### ISSUE TITLE
Make simulation comparison display faster 

### ISSUE THREAD

**Issue by poorva1209** [GRIDAPPSD/GOSS-GridAPPS-D#1581]

**Comment by ShuhaoBai** [comment]
Discovered a new issue that there is no stop mechanism for the current running simulation comparison plotting process. The comparison result is still being fetched even though user has closed the window, and it will not let the next submitted comparison start. 

Going to create a stop feature to fix this issue, in this task.

**Comment by ShuhaoBai** [comment]
The changes for this PR is included in https://github.com/gridappsd/goss-gridapps-d/issues/1641. Will create a new PR once 1641 is done.
